package.json

```diff
@@ -28,23 +28,23 @@
     "start": "npm run server:dev:hmr"
   },
   "dependencies": {
-    "@angular/http": "2.0.0-rc.4",
-    "@angular/common": "2.0.0-rc.4",
-    "@angular/compiler": "2.0.0-rc.4",
-    "@angular/core": "2.0.0-rc.4",
-    "@angular/forms": "0.2.0",
-    "@angular/platform-browser": "2.0.0-rc.4",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
-    "@angular/platform-server": "2.0.0-rc.4",
-    "es6-shim": "^0.35.0",
+    "@angular/common": "^2.0.0-rc.5",
```

Comment:
Good call. Looks like the [webpack starter](https://github.com/AngularClass/angular2-webpack-starter/blob/master/package.json) project most of this is derived from is using rc5 for those.